common/src/Polyhedron.h

```diff
@@ -230,12 +243,22 @@ class Polyhedron {
         V center() const;
         T intersectWithRay(const vm::ray<T,3>& ray, const vm::side side) const;
         vm::point_status pointStatus(const V& point, T epsilon = vm::constants<T>::pointStatusEpsilon()) const;
+
+        friend std::ostream& operator<<(std::ostream& stream, const Face& face) {
+            const auto* firstEdge = face.boundary().front();
+            const auto* currentEdge = firstEdge;
+            do {
+                stream << *currentEdge << std::endl;
+                currentEdge = currentEdge->next();
+            } while (currentEdge != firstEdge);
+            return stream;
+        }    private:
```

Comment:
the `private:` on this line can be deleted

Reply:
done